Assets/Scripts/Player/PlayerController.cs

```diff
@@ -62,14 +59,15 @@ public void SpawnBullet()
 
         public void DestroyPlayer()
         {
+            PlayerManager.Instance.playerSpawned -= InvokeEvents;
```

Comment:
why it is referring a service, instead there should be an event OnPlayerDestroy to which the playermanagere should listen